modules/i18n/src/translations/en-US/portals/myaccount.ts

```diff
@@ -190,7 +190,7 @@ export const myAccount: MyAccountNS = {
                             placeholder: "E.g. Starts with etc."
                         },
                         filterValue: {
-                            placeholder: "E.g. Zoom, Salesforce etc."
+                            placeholder: "Enter value to search"
```

Comment:
Shall we follow the same format and give example for the placeholder values, WDTY?

Reply:
Used same in Console ->applicaitons->i18n